src/test/java/dk/kb/present/TestFiles.java

```diff
@@ -28,6 +28,7 @@ public class TestFiles {
     public static final String PVICA_RECORD_e683b0b8 = "internal_test_files/tvMetadata/e683b0b8-425b-45aa-be86-78ac2b4ef0ca.xml";
     // White program radio metadata from 1966
     public static final String PVICA_RECORD_c295ae6c = "internal_test_files/tvMetadata/c295ae6c-fd34-4694-a9cc-4204b4a9f3a0.xml";
+    public static final String PVICA_RECORD_accf8d1c = "internal_test_files/tvMetadata/accf8d1c-3c4f-41a9-a1a2-48ce1cf0c5ca.xml";
```

Comment:
It's not a decent pull request if there hasn't been at least one addition or update to `aegis` :grin: